ansible/hosts

```diff
@@ -1,16 +1,21 @@
+#Sample Hosts File with variables
 
 [bootstrap]
-10.7.183.59
+192.168.0.1
 
 [master]
-10.7.183.6[0:1]
+192.168.0.[2:3]
 
 [workers]
-10.7.183.62
+192.168.0.4
 
 
 [bootstrap:vars]
 node_master=true
+boot_kube_version="v0.3.9"
+#API Server FQDN is for production deployments only. Can be used with an IP or hostname of a master server instead
+api_server_fqdn="kubeapi.test.local"
 
 [master:vars]
-node_master=true
\ No newline at end of file
+node_master=true
+cni_version="v0.4.0"
```

Comment:
Awesome.  I will update that to version 0.5.0, and re-test the deployment on Monday. Thanks!